### PR TITLE
[11.0][FIX] runbot buildout correct interreter

### PR DESCRIPTION
**Pull request by NL66278**
Prevent Odoo 11 (and higher) branches from being run by python2.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Seems OK. Code review

File: runbot_buildout/models/runbot_branch.py [hunk 1]
```diff
@@ -35,3 +39,21 @@ def _compute_buildout_version(self):
             if not match:
                 continue
             this.update({'buildout_version': match.groupdict()['version']})
+
+    @api.multi
+    def get_interpreter(self):
```
Comment (moylop260) [inline runbot_buildout/models/runbot_branch.py:44]: What about get interpreter from magic comment of odoo-bin file?

Comment (hbrunn) [reply runbot_buildout/models/runbot_branch.py:44]: that doesn't exist, repos that rely on buildouts never have any plain odoo/ocb repo in runbot, just the buildout and the code/module branches

File: runbot_buildout/models/runbot_build.py [hunk 2]
```diff
@@ -414,7 +415,7 @@ def _adapt_buildout(self):
                 'options.without_demo = False\n'
                 'options.data_dir = %(datadir)s\n'
                 'options.dbfilter = %(dbfilter)s\n'
-                'options.lang = en_US' % dict(
+                'options.load_language = en_US' % dict(
```
Comment (hbrunn) [inline runbot_buildout/models/runbot_build.py:418]: see https://github.com/OCA/runbot-addons/pull/187, but keep it here, then I'll close mine

File: runbot_buildout/models/runbot_branch.py (hunk 1 above)
Comment (NL66278) [reply runbot_buildout/models/runbot_branch.py:44]: @moylop260 we need the interpreter to bootstrap the buildout. This bootstrap wil create the bin/buildout script that, after running the buildout, will create the odoo part that contains odoo-bin.

File: runbot_buildout/models/runbot_build.py (hunk 2 above)
Comment (NL66278) [reply runbot_buildout/models/runbot_build.py:418]: @hbrunn It got mixed up/in because my starting point was your branch. Now merged anyway.

**Pull request merged**